cookbooks/fb_systemd/recipes/default.rb

```diff
@@ -93,6 +93,11 @@
       map { |x| " --exclude-prefix=#{x}" }.
       join
   }
+  # it returns 65 if it had to ignore some lines, which seems to happen
+  # quite often on initial setup
+  if node.firstboot_any_phase?
+    returns [0, 65]
+  end
```

Comment:
Should we warn something if it's skipping lines?

Reply:
there's no good way to behave differently on different return values unless we want to write our own resource. In this case, since it's only on firstboot, I figure it's not worth it.